src/Microsoft.PowerShell.Commands.Management/resources/ProcessResources.resx

```diff
@@ -132,6 +132,9 @@
   <data name="ProcessNameForConfirmation" xml:space="preserve">
     <value>{0} ({1})</value>
   </data>
+  <data name="ProcessStartInfo" xml:space="preserve">
+    <value>{0} {1}</value>
+  </data>
```

Comment:
We re-use `ProcessNameForConfirmation` in Stop-Process and Debug-Process so I believe we can use the same in Start-process.

Reply:
The original request was to output the exact command being executed. By re-using the ProcessNameForConfirmation, the format of the output will be changed slightly, the arguments would appear in the parentheses. i.e. `C:\WINDOWS\system32\PING.EXE (-n 2 localhost)` instead of `C:\WINDOWS\system32\PING.EXE -n 2 localhost`.
If that is acceptable, I'm happy to make the change.

Reply:
Thanks for clarify.
Closed.